examples/petsc/src/matops.c

```diff
@@ -6,29 +6,34 @@
 // -----------------------------------------------------------------------------
 PetscErrorCode MatGetDiag(Mat A, Vec D) {
   PetscErrorCode ierr;
-  UserO user;
+  OperatorApplyContext op_apply_ctx;
 
   PetscFunctionBeginUser;
 
-  ierr = MatShellGetContext(A, &user); CHKERRQ(ierr);
+  ierr = MatShellGetContext(A, &op_apply_ctx); CHKERRQ(ierr);
 
   // Compute Diagonal via libCEED
-  PetscScalar *y;
+  PetscScalar *x;
```

Comment:
We should be using `y` here, not `x`. These changes are undoing https://github.com/CEED/libCEED/pull/1075

Reply:
Yeah, that is mixed-up during the rebase.